frame/contracts/src/tests.rs

```diff
@@ -5633,3 +5633,27 @@ fn root_cannot_instantiate() {
 		);
 	});
 }
+
+#[test]
+fn sign_extension_works() {
+	let (code, _) = compile_module::<Test>("sign_extension").unwrap();
+
+	ExtBuilder::default().build().execute_with(|| {
```

Comment:
this fails with code rejected when sign_extension is false

Reply:
how about adding some assertion to the test which also checks that the [sign-extension](https://github.com/WebAssembly/spec/blob/main/proposals/sign-extension-ops/Overview.md) operator (`i32.extend8_s` in your fixture) produces the right value?

Reply:
actually moved the test to wasm/prepare, seems like a more appropriate place for this test. I don't think we should test the output of the wasm ops (unless you want to be exhaustive and verify all of them)